src/r2mm/installing/profile_installers/ModLoaderVariantRecord.ts

```diff
@@ -47,4 +47,5 @@ export const MOD_LOADER_VARIANTS: {[key: string]: ModLoaderPackageMapping[]} = {
     Titanfall2: [new ModLoaderPackageMapping("Northstar-Northstar", "Northstar", PackageLoader.NORTHSTAR)],
     Peglin: [new ModLoaderPackageMapping("BepInEx-BepInExPack_Peglin", "BepInExPack_Peglin", PackageLoader.BEPINEX)],
     VRising: [new ModLoaderPackageMapping("BepInEx-BepInExPack_V_Rising", "BepInExPack_V_Rising", PackageLoader.BEPINEX)],
+    MinutesTillDawn: [new ModLoaderPackageMapping("BepInExPackMTD-BepInExPack_20MTD", "BepInExPack_20MTD", PackageLoader.BEPINEX)],
```

Comment:
This would also need changing

Reply:
Another thing to note is that I can upload a package under the BepInEx team if preferred. The BepInExPackMTD team isn't necessarily needed.

Reply:
> Another thing to note is that I can upload a package under the BepInEx team if preferred. The BepInExPackMTD team isn't necessarily needed.
Would that allow that the community to make changes to the BepInExPack without having to bother you every time?
In that case I'd happily agree. Otherwise I think it might be better if we can change it for ourselves as we're still early in deciding things. We don't have a base library yet but that could change.